R/lognormal_ab_test.R

```diff
@@ -1,104 +1,130 @@
-lognormal_ab_test <- function(data, nsim=1e5, alpha0=1, beta0=25, m0=4, k0=1, s_sq0=1, v0=5, expected_revenue_converted_users=NULL, expected_conversion_rate=NULL, plot.density=FALSE, conf.level=.1, tolerance=.01, save.hist=TRUE){  
-  library(ggplot2)
-  #bayes.lognormal.test <- function(data, nsim=1e5, alpha0=1, beta0=1, m0=4, k0=1, s_sq0=1, v0=1, plot.density=FALSE, conf.level=.1, tolerance=.01){  
-  
-  # probability model:  
-  # conversion - binomial likelihood, beta prior with shape parameters alpha0, beta0  
-  # revenue - Normal likelihood model for log(revenue). Prior: 
+lognormal_ab_test <- function(data,
+                              nsim = 1e5,
+                              alpha0 = 1,
+                              beta0 = 25,
+                              m0 = 4, k0 = 1, s_sq0 = 1, v0 = 5,
+                              expected_revenue_converted_users = NULL,
+                              expected_conversion_rate = NULL,
+                              plot.density = FALSE,
+                              conf.level = 0.1,
+                              tolerance = 0.01,
+                              save.hist = TRUE) {
+  # Probability model:
```

Comment:
This is totally not finished yet. @ayakubovich to be candid, I need to go through this and teach myself what you're doing here.  I can see that you're using this to test for differences in revenue, but at the moment it's just code to me.  Do you have any sources handy?